Update the introduction (#2922)
exercism/java: concepts/basics/introduction.md

# Introduction

Java is a statically-typed language, which means that the type of a variable is known at compile-time. Assigning a value to a name is referred to as defining a variable. A variable is defined by explicitly specifying its type.

```java
int explicitVar = 10;
```

> Updating a variable's value is done through the `=` operator. Here, `=` does not represent mathematical equality. It simply assigns a value to a variable. Once defined, a variable's type can never change.

Reviewer: I think it might more friendly if we just explain what `=` does (not sure if the "mathematical equality" will be understood by everyone).  

```suggestion
The `=` operator assigns a value to a variable. It can also be used to update a variable's value.
```

}
```

Invoking/calling a method is done by specifying its class and method name and passing arguments for each of the method's parameters.

```java

> int sum = new Calculator().add(1, 2);  // here the  "add" method has been called to perform the task of addition 

Reviewer: This suggestion is mainly to reduce the width (reducing chance of needing side scrolling)

```suggestion
// call "add" method to perform addition 
int sum = new Calculator().add(1, 2);  
```
